src/_data/team.yml

```diff
@@ -850,4 +849,17 @@
     <p> In his free time he enjoys making and listening to music, playing trombone in his band and spending time with family, friends and especially his 3 year old son.</p>
   github: mattgraygithub
   email: matthew.gray@codurance.com
+  
+- id: tacuma_bellford
+  name: Tacuma Bellford
+  image: /assets/custom/img/team/tacuma_bellford.jpg
+  role: Apprentice
+  fullDescription:
+    <p>Tacuma’s background is in Linguistics and enjoys learning languages, for machines and humans. Computational Linguistics, is what sparked his interest in software development. His motivation comes from the thought of being able to bring ideas to life using technology.</p>
+    <p>His professional career started with a startup solving a problem in the medical education sector. He was involved in the front end aspect of development and also in some decisions regarding the direction of the platform.</p>
```

Comment:
Where you have 'career started with a startup', seems to me that it would read better as 'career began with (or at) a startup'.

Reply:
Thanks Keith. I will fix that.

Reply:
This is all good.